app/upgrades/v19/upgrades.go

```diff
@@ -36,10 +42,19 @@ func CreateUpgradeHandler(
 		ctxCache, writeFn := ctx.CacheContext()
 		if err := RemoveOutpostsFromEvmParams(ctxCache, ek); err == nil {
 			writeFn()
+		} else {
+			logger.Debug("Error removing outposts.")
```

Comment:
@0xstepit for future reference, it is best practice in golang to have the errors be lowercase for consistency

```suggestion
			logger.Debug("error removing outposts")
```